integrations/ScriptLoader.js

```diff
@@ -9,10 +10,16 @@ const ScriptLoader = (id, src, async = defaultAsyncState) => {
   js.async = async === undefined ? defaultAsyncState : async;
   js.type = "text/javascript";
   js.id = id;
-  const e = document.getElementsByTagName("script")[0];
-  logger.debug("==parent script==", e);
-  logger.debug("==adding script==", js);
-  e.parentNode.insertBefore(js, e);
+  const headElems = document.getElementsByTagName("head");
+  if (Object.keys(headElems).length !== 0) {
+    logger.debug("==adding script==", js);
+    headElems.appendChild(js);
```

Comment:
We should not append it but rather insert it before the first child (it needn't be a script element)

Reply:
updated. Please take a look if its okay.